### PR TITLE
Passes ENV along to child processes

### PR DESCRIPTION
**Pull request by elorest**
### Description of the Change

Fixes https://github.com/amberframework/amber/issues/619

Passes ENV module along to child processes after turning it into a hash so that a couple other params could be merged in.

Please suggest effective way to write tests for this, since specs can't set ENV vars outside of process.

### CODE REVIEW

File: src/amber/server/server.cr [hunk 1]
```diff
@@ -43,9 +43,10 @@ module Amber
 
     def run
       thread_count = settings.process_count
+      env_hash = ENV.to_h
       if Cluster.master? && thread_count > 1
         while (thread_count > 0)
-          Cluster.fork ({"id" => thread_count.to_s})
+          Cluster.fork (env_hash.merge({"id" => thread_count.to_s}))
```
Comment (eliasjpr) [inline src/amber/server/server.cr]: Can we do this in the cluster class instead?

Comment (eliasjpr) [reply src/amber/server/server.cr]: And I think crystal classes has a dup/clone method see https://github.com/amberframework/amber/blob/3374da08814340e50c4deb47daeaa8cb785311d8/src/amber/server/cluster.cr#L5

**Comment by eliasjpr** [comment]
Also I don’t recommend to use merge since is appending to the env object an is key

**Comment by elorest** [comment]
I don't understand what you mean by:

> Also I don’t recommend to use merge since is appending to the env object an is key

File: src/amber/server/server.cr [hunk 2]
```diff
@@ -45,7 +45,7 @@ module Amber
       thread_count = settings.process_count
       if Cluster.master? && thread_count > 1
         while (thread_count > 0)
```
Comment (drujensen) [inline src/amber/server/server.cr]: WDYT?
```
(0...thread_count).each do |cnt|
  Cluster.fork
end
```

File: src/amber/server/server.cr [hunk 3]
```diff
@@ -44,10 +44,7 @@ module Amber
     def run
       thread_count = settings.process_count
       if Cluster.master? && thread_count > 1
-        while (thread_count > 0)
-          Cluster.fork ({"id" => thread_count.to_s})
-          thread_count -= 1
-        end
+        thread_count.times{ Cluster.fork }
```
Comment (faustinoaq) [inline src/amber/server/server.cr]: Small formatting issue here: _missing space_ `thread_count.times { Cluster.fork }`

Comment (elorest) [reply src/amber/server/server.cr]: I ran `crystal tool format` on it before committing and this is how it made it...

 It looks very strange to me to put a space between the method and `()` and `{}`? When did we start doing that?

Comment (faustinoaq) [reply src/amber/server/server.cr]: I think is just a convention, by default `crystal tool format` _adds_ an space in between `method {` (braces) and _ignore_ spaces between `method (` (parenthesis)

Comment (faustinoaq) [reply src/amber/server/server.cr]: But anyway, this PR looks good, let's merge it 👍

**Pull request merged**